valet-key/README.md

```diff
@@ -0,0 +1,115 @@
+---
+layout: pattern
+title: Valet Key
+folder: valet-key
+permalink: /patterns/valet-key/
+categories: Behavioral
+tags:
+ - Data access
```

Comment:
Valet key is a cloud pattern and I'd preferably like to see it implemented using cloud technologies. One cloud pattern example is at https://github.com/iluwatar/java-design-patterns/tree/master/cloud-static-content-hosting. 

In the example implementation this article series might also be interesting: https://www.keepsecure.ca/blog/aws-vs-azure-object-storage-valet-key-pattern-part-1/

Reply:
Thank for your advice. But it is my first time to use cloud technology, I may take more to finish it.

Reply:
No problem @EdisonE3, take your time. I think learning cloud technologies is an excellent investment in the future.

Reply:
Can I use a local server to replace a real cloud server? I think use local server will be more easy to be tested and if there is necessary, users can deploy it to the Cloud by themselves. @iluwatar

Reply:
@EdisonE3 cloud patterns need to be implemented using cloud technologies. See https://github.com/iluwatar/java-design-patterns/wiki/10.-Technology-selections